roosevelt.js

```diff
@@ -169,6 +171,16 @@ module.exports = function (params) {
     params.onServerInit(app)
   }
 
+  // use user-defined cssCompiler
+  if (params.cssCompiler && typeof params.cssCompiler === 'function') {
+    cssProcessor = params.cssCompiler(app)
```

Comment:
this work should be done in sourceParams instead of doing it here and then passing it as an argument to the compiler modules

the individual compiler modules can get the compiler out of params later